xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -0,0 +1,694 @@
+/*
+ *      Copyright (C) 2016 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "ContextMenuItem.h"
+#include "cores/AudioEngine/Engines/ActiveAE/AudioDSPAddons/ActiveAEDSP.h"
+#include "epg/EpgInfoTag.h"
+#include "pvr/addons/PVRClients.h"
+#include "pvr/channels/PVRChannel.h"
+#include "pvr/PVRGUIActions.h"
+#include "pvr/PVRManager.h"
+#include "pvr/recordings/PVRRecording.h"
+#include "pvr/recordings/PVRRecordingsPath.h"
+#include "settings/Settings.h"
+#include "pvr/timers/PVRTimers.h"
+#include "utils/URIUtils.h"
+
+#include "PVRContextMenus.h"
```

Comment:
Ha, we have something already... => https://codedocs.xyz/xbmc/xbmc/code_guidelines.html#code_guidelines_1_3

<pre>
Included header files *.h, are to sort alphabetical to prevent double used file definition and allow better overview

    On cpp define used header first to confirm needed headers are present on them itself
    As next use the global system headers e.g. <cassert> or "system.h"
    Then insert all the headers needed for the file itself
</pre>

Reads like the Google guideline (though not formulated clear enough).